### ISSUE TITLE
No module named fedora_cert

### ISSUE THREAD

**Issue by bfallik** [pypingou/fedora-active-user#11]
Tried to use this script on OSX and hit this failure:
```
$ ./fedora_active_user.py --help
Traceback (most recent call last):
  File "./fedora_active_user.py", line 25, in <module>
    import fedora_cert
ImportError: No module named fedora_cert
```

**Comment by pypingou** [comment]
This is part of the fedora-cert package which is upstream in https://pagure.io/fedora-packager

**Comment by bfallik** [comment]
ah, thx. What do you think about adding some details to the README about dependencies to use this script?

**Comment by pypingou** [comment]
Sure, that sounds good, do you want to do it?

**Comment by bfallik** [comment]
On Fri, Mar 31, 2017 at 11:02 AM, Pierre-Yves Chibon <
notifications@github.com> wrote:

> Sure, that sounds good, do you want to do it?
>

Sure.

**Comment by bfallik** [comment]
PR in https://github.com/pypingou/fedora-active-user/pull/12.

**Comment by pypingou** [comment]
Fixed in PR #12
